<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librarianWindow</class>
 <widget class="QWidget" name="librarianWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hinlibs [librarian]</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>600</width>
     <height>640</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="profile_librarian">
    <widget class="QWidget" name="header" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>600</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #A3D5FF</string>
     </property>
     <widget class="QToolButton" name="ProfileButtonInProfileLibrarian">
      <property name="geometry">
       <rect>
        <x>535</x>
        <y>15</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Profile</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/avatar-icon.svg</normaloff>:/icons/avatar-icon.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>33</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="filterWidget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #EEE;</string>
     </property>
     <widget class="QPushButton" name="logOutButtonLibrarian">
      <property name="geometry">
       <rect>
        <x>515</x>
        <y>5</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="welcomeUserLabelLibrarian">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Librarian Dashboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addItemButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item to Catalogue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeItemButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>221</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Item From Catalogue</string>
     </property>
    </widget>
    <widget class="QPushButton" name="returnItemButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>460</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Return Item for Patron</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageAddItem">
    <widget class="QWidget" name="header_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #A3D5FF</string>
     </property>
     <widget class="QToolButton" name="BackFromAddItem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Go back</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/back-icon.svg</normaloff>:/icons/back-icon.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="welcomeUserLabelLibrarian_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Add Item To Catalogue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>511</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="authorCreatorLabel">
        <property name="text">
         <string>Author / Creator :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="authorEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="formatCombo">
        <item>
         <property name="text">
          <string>Book</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magazine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Movie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>VideoGame</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="publicationYearLabel">
        <property name="text">
         <string>Publication Year:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="yearSpin">
        <property name="minimumSize">
         <size>
          <width>389</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>2100</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="genreLabel">
        <property name="text">
         <string>Genre:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="genreEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="deweyDecimalLabel">
        <property name="text">
         <string>Dewey decimal:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="deweyEdit"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ratingLabel">
        <property name="text">
         <string>Rating:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="ratingEdit"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="issueNumberLabel">
        <property name="text">
         <string>Issue number:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="issueEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="addItemConfirmButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>470</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageRemoveItem">
    <widget class="QWidget" name="header_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #A3D5FF</string>
     </property>
     <widget class="QToolButton" name="BackFromRemoveItem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Go back</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/back-icon.svg</normaloff>:/icons/back-icon.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="welcomeUserLabelLibrarian_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Remove Item From Catalogue&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="removeItemIdEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter Item ID (e.g. I012)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeItemConfirmButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Item</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageReturnOnBehalf">
    <widget class="QWidget" name="header_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #A3D5FF</string>
     </property>
     <widget class="QToolButton" name="BackFromReturnItem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>50</width>
        <height>50</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Go back</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #FFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/icons/back-icon.svg</normaloff>:/icons/back-icon.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="welcomeUserLabelLibrarian_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>600</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFF</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Return Item For Patron&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Patron Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="returnPatronUsernameEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="findPatronButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Find Patron</string>
     </property>
    </widget>
    <widget class="QLabel" name="returnPatronStatusLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="returnLoansTable">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="returnSelectedItemButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>490</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Return Selected Item</string>
     </property>
    </widget>
    <widget class="QLabel" name="returnItemResultLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>540</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
